### ISSUE TITLE
FASTCALL is in cpython 3.6, and cpython 3.6 won't change

### ISSUE THREAD

**Issue by illume** [pygame/pygame#3527]
When "FASTCALL" was added it checks for python 3.7. https://github.com/pygame/pygame/pull/3325

It would make sense to use it with 3.6 because cpython 3.6 is not going to remove support for FASTCALL.
Am pretty sure, but haven't double checked, that other modules are using it in 3.6 too.

**Comment by Starbuck5** [comment]
I believe we use FASTCALL on Pypy as well (emulated of course, so not for a speedup), and pypy 3.6 did not support FASTCALL emulation. Also we were happy to stick to what the Python docs say, maybe FASTCALL wasn't fully complete in 3.6.

So it's a bit more complicated than us doing a redundant check. We'd need to replace it with another check.

But 3.6 is end of life anyways, I'd like to drop it after 2.1.3.

**Comment by ankith26** [comment]
Yes, my main reasoning for adding the 3.7 check is that fastcall cpython 3.6 was first of all, and implementation detail (and other python implementations don't have it), and more importantly, it also did not exactly match 3.7+ fastcall API AFAIK (some differences in how keywords are handled and such). Of course, with more code specialcasing cpython 3.6 we can technically make it use fastcall as well, but I think it's not worth the effort because it's just a performance thing that's not going to break anyone's code running on python 3.6
